lib/src/dom/accelerometer.dart

```diff
@@ -30,11 +30,16 @@ extension AccelerometerExtension on Accelerometer {
 
 @JS('AccelerometerSensorOptions')
```

Comment:
nit: You don't have to address this here, but we don't need this renaming for two reasons:

1. It's the same as the class name.
2. It's an @anonymous class so you don't even have a named constructor anymore.